tapdb/supply_ignore_checker.go

```diff
@@ -3,8 +3,10 @@ package tapdb
 import (
 	"context"
 	"errors"
+	"fmt"
```

Comment:
Re the commit message: do we need a supply verifier to invalidate it? Couldn't we just invalidate it selectively when/if an asset in that group is ignored?